lib/elixir/src/elixir_erl_try.erl

```diff
@@ -86,7 +86,7 @@ erl_rescue_stacktrace_for(_Meta, _Var, 'Elixir.ErlangError') ->
   error(badarg);
 erl_rescue_stacktrace_for(Meta, Var, 'Elixir.KeyError') ->
   %% Only the two element tuple requires stacktrace.
-  erl_and(Meta, erl_tuple_size(Meta, Var, 2), erl_record_compare(Meta, Var, badkey));
+  erl_and(Meta, erl_tuple_size(?generated(Meta), Var, 2), erl_record_compare(Meta, Var, badkey));
```

Comment:
I need to test with the other versions of Erlang, but it seems that only tuple_size that is causing warnings is this one, even with other types of exception

Reply:
I tried to add the minimum quantity of generated, but I think we would also make all the metas in `erl_rescue_stacktrace_for` were generated

Reply:
I think that when we call `erl_rescue_stacktrace_for` in the first place, we can already pass a ?generated(Meta), so everything is marked as generated from the beginning, across all clauses. What do you think?

Reply:
Agreed